stake-pool/js/package.json

```diff
@@ -1,40 +1,22 @@
 {
```

Comment:
Why all the changes to `package.json` and `package-lock.json`? From first glance, it doesn't seem like most of these changes are necessary (other than maybe the `lint` script and possibly `main`). The conflicting changes to these files might make it a little more difficult to merge with the #2804 PR (which was coincidentally made a few hours before this one), and a lot of the removals don't really make sense for me.